Chewbawka2017/src/org/team1540/chewbawka/Shooter.java

```diff
@@ -1,7 +1,92 @@
 package org.team1540.chewbawka;
 
-public class Shooter {
-	public static void setup() {
+import ccre.channel.BooleanInput;
+import ccre.channel.EventInput;
+import ccre.channel.FloatCell;
+import ccre.channel.FloatInput;
+import ccre.channel.FloatOutput;
+import ccre.cluck.Cluck;
+import ccre.ctrl.ExtendedMotor.OutputControlMode;
+import ccre.ctrl.ExtendedMotorFailureException;
+import ccre.ctrl.StateMachine;
+import ccre.drivers.ctre.talon.TalonExtendedMotor;
+import ccre.frc.FRC;
 
+public class Shooter {
```

Comment:
Liam's shooter code is generally prettier and cleaner. Take a look at his code; PIDTalon, .selectByState(), etc. I'd recommend you take a look.